projects/components/src/multi-select/multi-select.component.ts

```diff
@@ -249,6 +257,11 @@ export class MultiSelectComponent<V> implements ControlValueAccessor, AfterConte
     this.setTriggerLabel();
   }
 
+  public onPopoverOpen(): void {
+    this.popoverOpen = true;
+    this.reorderSelectedItemsSubject.next(true);
```

Comment:
we do not need this subject. can be removed. so, by extension, this method also isn't required.

Reply:
I moved it to the Close